src/core/services/gesture/gesture.spec.js

```diff
@@ -688,4 +688,36 @@ describe('$mdGesture', function() {
 
   });
 
+  describe('contenteditable', function() {
+
+    // Click tests should only be enabled when `$$hijackClicks == true` (for mobile)
+
+    it('should not hijack click on contenteditable element', inject(function($document, $mdGesture) {
+      if ( $mdGesture.$$hijackClicks ) {
```

Comment:
Hmm... you're right.  I used one of the existing tests as a template, but apparently that property no longer exists.  I'll make the correction.